backend/configs/main.conf

```diff
@@ -4,13 +4,17 @@ IPCameraTimeOut=2000 #time the values are saved of an ip camera
 camera_1_type=simplecamera
 camera_1_address=http://tuincam.bt.tudelft.nl/mjpg/video.mjpg
 camera_1_macaddress=simplecamera1
+camera_1_aspectratio=1.7777
+
 camera_2_type =simplecamera
 camera_2_address=http://131.180.123.51/zm/cgi-bin/nph-zms?mode=jpeg&monitor=2&scale=100&buffer=100
 camera_2_macaddress=simplecamera2
+camera_2_aspectratio=1.7777
 
-camera_3_type=simplecamera
-camera_3_address=http://delficam.ewi.tudelft.nl/axis-cgi/mjpg/video.cgi?camera=1&resolution=704x576&1464863399284
-camera_3_macaddress=simplecamera3
+#camera_3_type=simplecamera
```

Comment:
I don't think we want to merge this commented version of the "default" (which is not officially default) version into develop.